packages/lavapack/src/runtime.js

```diff
@@ -6423,13 +6423,13 @@ function makePrepareRealmGlobalFromConfig ({ createFunctionWrapper }) {
     // verify + load config
     if (bundlePolicy) loadPolicy(bundlePolicy)
     // verify + load in each module
-    for (const [moduleId, moduleDeps, initFn, { package: packageName }] of newModules) {
+    for (const [moduleId, moduleDeps, initFn, { package: packageName, type }] of newModules) {
       // verify that module is new
       if (moduleRegistry.has(moduleId)) {
         throw new Error(`LavaMoat - loadBundle encountered redundant module definition for id "${moduleId}"`)
       }
       // add the module
-      moduleRegistry.set(moduleId, { type: 'js', id: moduleId, deps: moduleDeps, source: `(${initFn})`, package: packageName })
+      moduleRegistry.set(moduleId, { type: type || 'cjs', id: moduleId, deps: moduleDeps, source: `(${initFn})`, package: packageName })
```

Comment:
still cjs here

Reply:
Oops, forgot to rebuild the runtime, done now :+1: